### ISSUE TITLE
Window.Show Cannot Accept Null

### ISSUE THREAD

**Issue by mysteryx93** [AvaloniaUI/Avalonia#9155]
Window.Show has 2 overloads, without or without parent.

The Parent argument should thus be optional and nullable, but it's not.

https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/Window.cs#L609

If I don't know whether I'll have a owner, it thus requires me to write this

```c#
if (Owner != null)
{
    Ref.Show(Owner);
}
else
{
    Ref.Show();
}
```

- Version 11.0.0-preview2

**Comment by timunie** [comment]
> The Parent argument should thus be optional and nullable, but it's not.

I don't think so tbh. One function is for showing a window as a dependent child window, the other is to show a window without a parent. I think the developer should have a control over what is intended. It's probably not a good idea if Avalonia "guesses" it on it's own. 

So I think the 6 lines of code you show above are totally valid and needed. You can make an extension method if you want to re-use it. 

Does it makes sense to you?
